Day 2 - Python Datetime objects and Timedelta/Q2_TimeDifference.py

```diff
@@ -0,0 +1,15 @@
+# 2. What is the time difference between January 1, 2019 11 AM
+#    and May 20, 2020 22.30 PM? Mention both the days and time
+#    in your solution.
+
+from datetime import datetime, timedelta
+
+future_date = datetime(year=2020, month=5, day=20, hour=22, minute=30)
+old_date = datetime(year=2019, month=1, day=1, hour=11)
+
+time_difference = future_date - old_date
+
+print("Difference in days.. ", (time_difference.days*24 + time_difference.seconds/3600)/24)
```

Comment:
Your approach is correct. But your answer is providing a decimal number. 
In the question, it asks to provide the time difference. Then it says mention both the days and time. Which means, you should provide your answer like "Time difference is x days y hours z mins...so on". Is it clear?

Reply:
Comments addressed!

Reply:
LGTM !!